### PR TITLE
setup initial App component with fetch weather data

### PR DESCRIPTION
**Pull request by artanmyrtolli**
What does this PR do?

Set up App and fetch call with NWS data, as well as use React Context to pass the data to a new component. 

What (if any) features are you implementing?

 [] Bug fix (non-breaking change which fixes an issue)
 [] Refactor (code is working and without bugs, formatting has been streamlined)
 [x] New feature (non-breaking change which adds functionality)
 [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

[x] Have you used Dev Tools/ ran code in the console?
[x] Have you verified changes using webpack?

Code review:

Are the commit messages properly formatted?
Are the branch names properly formatted? Are they consistent with the naming convention we determined as a group?
Are the variables named properly and clearly in this code?
(Arrays are plural, etc)
Are functions in this code named properly?
(Starts with an action verb, describes what the function does)
Are any of the functions super long or doing too much? Can any parts be broken out into smaller, more modular helper functions?
Are there any parts of this code that I don't understand and need to review on a call with my partner?

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,40 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { Component, createContext } from 'react';
+import WeatherContainer from './Components/WeatherContainer/WeatherContainer';
```
Comment (laytonmaes) [inline src/App.js:3]: neat

**Review by nhodnett — APPROVED** [review]
All good. Nice work team!

**Review by laytonmaes — APPROVED** [review]

**Review by King13k — COMMENTED** [review]
Good job team getting the Context Api to work!

**Review by King13k — APPROVED** [review]
Neato

**Pull request merged**